tools/net-common.in

```diff
@@ -63,6 +63,41 @@ function fix_mac {
   ip link set $INTERFACE address $FIXED_MAC || true
 }
 
+function bridge_vlan_add {
+  local VID="${1}"
+  shift
+  local EXTRA="${@}"
+  # add the VLAN to the tap
+  bridge vlan add dev ${INTERFACE} vid ${VID} ${EXTRA}
```

Comment:
`$@` is treated specially by most shells (at least bash and dash). From `bash(1)`:

> When the expansion [of $@] occurs within double  quotes,  each parameter  expands  to  a separate word.

By using "${EXTRA}" here we are losing that "special" expansion, whose purpose is to add to safety. I would prefer using `"$@"` directly in bridge the command.